packages/flame/src/Alert/AlertInCard.tsx

```diff
@@ -42,9 +43,11 @@ const AlertInCard: React.FC<Props> = ({
       <Flex className="fl-alert__icon" flex={0} mt="-1px">
         <AlertIcons type={type} />
       </Flex>
-      <div css={css({ flex: '1', pl: 2, fontSize: ['text', 'text-s'] })}>{children}</div>
+      <Text css={css({ flex: '1' })} fontSize={['text', 'text-s']} lineHeight={[3, 2]} pl={2}>
+        {children}
+      </Text>
       {!noCloseBtn && (
-        <Flex css={css({ flex: '0' })}>
+        <Flex flex={0}>
```

Comment:
Out of scope comment: eventually we'll use Styled System's `css` under the hood for those prop, and dissallow `css` completely for all components and restrain flexibility, correct?

Reply:
yes and no

Internally in the library, it's fine. I don't care if we have `css` or styled-system props within the library. But users of the library shouldn't have to override or use these properties if we ensure the components to do all the interactions properly.

Whenever we see a proliferation of props and overrides, it probably means that there's something in the component behaviour that do not sync up with what is expected of them. I'd prefer seeing spacer components instead of having a bunch of random css/spacing properties, as its more declarative and restrictive, which ensure that we have *some* consistency in UI

You could still have some styled-system props baked into a set of particular components. Box/Flex/Text are good candidates for this since these are kinda "container" components. So having control over spacing and some flex properties is nice. What will be disallowed are things like setting border color and stuff that goes beyond simple positioning.

As for our other components (button, card and so on) for consumers: no spacing props, styled system props.

If an override or adjustment is needed, they can leverage css props or we can provide a way to inject their custom components if need be.